src/alexapi/device_platforms/windowsplatform.py

```diff
@@ -0,0 +1,37 @@
+import logging
```

Comment:
It was just to do `if not self.__config['platform']['device'] == 'windows':` but i will do with `platform.system()` so it's no more needed